frontend/src/components/maps/leaflet/LayerPopup/LayerPopupContent.tsx

```diff
@@ -65,22 +82,30 @@ export const LayerPopupContent: React.FC<IPopupContentProps> = ({
           </ListGroup.Item>
         ))}
       </ListGroup>
-      {populateDetails && (data.PID !== undefined || data.PIN !== undefined) ? (
-        <Link
-          style={{ display: 'flex', justifyContent: 'center', paddingBottom: '10px' }}
-          onClick={(e: any) => onAddToParcel(e, { ...data, CENTER: center })}
-          to={{
-            search: queryString.stringify({
-              ...queryString.parse(location.search),
-              sidebar: true,
-              disabled: false,
-              loadDraft: false,
-            }),
-          }}
-        >
-          Populate property details
-        </Link>
-      ) : null}
+      <MenuRow>
+        <Col>
+          {populateDetails && (data.PID !== undefined || data.PIN !== undefined) ? (
+            <StyledLink
+              onClick={(e: any) => onAddToParcel(e, { ...data, CENTER: center })}
+              to={{
+                search: queryString.stringify({
+                  ...queryString.parse(location.search),
+                  sidebar: true,
+                  disabled: false,
+                  loadDraft: false,
+                }),
+              }}
+            >
+              Populate property details
+            </StyledLink>
+          ) : null}
+          {bounds && curZoom! !== boundZoom ? (
+            <StyledLink to={{ ...location }} onClick={() => leaflet.map?.flyToBounds(bounds)}>
```

Comment:
this flyToBounds sometimes goes all the way to zoom level 18, but for the buildings and parcels it only goes to 14. Do we want this to stop at 14 as well?

Reply:
Not sure, we'll test and get feedback